src/API/auth/facebook/passport.js

```diff
@@ -1,28 +1,41 @@
-import passport from 'passport';
-import passportFacebook from 'passport-facebook';
+/* istanbul ignore next */
+// Should be ignored for now because of third party dependency. So API tests can't help
 
-const FacebookStrategy = passportFacebook.Strategy;
+import passport from 'passport';
+import FacebookTokenStrategy from 'passport-facebook-token';
 
 /**
  * @function setup
  * @description Register facebook authentication strategy
  * @param {Object} UserService
- * @param {Object} config
+ * @param {Object} facebookConfig
  */
-export default function setup(UserService, config) {
-    passport.use(new FacebookStrategy({
-        clientID: config.facebook.clientID,
-        clientSecret: config.facebook.clientSecret,
-        callbackURL: config.facebook.callbackURL
+export default function setup(UserService, facebookConfig) {
```

Comment:
Yes, but I think injection is create some borders :). And it should help when we start write unit tests.